.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  test-python-versions:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: nanasess/setup-chromedriver@master
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: "14"
+      - name: Use Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Python Dependencies
+        run: pip install -r requirements/test-run.txt
+      - name: Run Tests
+        run: |
+          npm install -g npm@v7.13.0
```

Comment:
I'm not sure why I pinned the version of NPM here. I'd rather get an error when something no longer works with the latest version of NPM. So if we could just swap this line out for the following that would be great:

```
npm install -g npm@latest
npm --version
```